packages/app-check/src/internal-api.test.ts

```diff
@@ -284,6 +284,67 @@ describe('internal api', () => {
       );
       expect(token).to.deep.equal({ token: fakeRecaptchaAppCheckToken.token });
     });
+
+    it('reads any memory-cached debug token if in debug mode', async () => {
+      const clock = useFakeTimers();
+      const clientStub = stub(client, 'exchangeToken');
+      const readStub = stub(storage, 'readTokenFromStorage');
+      const debugState = getDebugState();
+      debugState.enabled = true;
+      debugState.token = new Deferred();
+      debugState.token.resolve('my-debug-token');
+      debugState.exchangeToken = fakeCachedAppCheckToken;
```

Comment:
`exchangeToken` doesn't sound obvious to me what it is. And having another field `token` confuses me even more. 
Should we change them to `appChekcToken` and `debugToken` respectively?

Reply:
Avoided separating them per comment below.